### PR TITLE
Mayhem initial integration

### PR DESCRIPTION
**Pull request by code-terror**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+project: pytils
```
Comment (xansec) [inline Mayhemfile:1]: Cmd needs to be updated to just the name of the python file to avoid fuzzing python.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 1 above)
Comment (code-terror) [reply Mayhemfile:1]: @xansec tried omitting python...The analysis fails..

Comment (xansec) [reply Mayhemfile:1]: You will need to handle your exceptions:

```
 === Uncaught Python exception: ===
ValueError: Value must be positive or zero, not -84
Traceback (most recent call last):
  File "fuzz-numeral.py", line 9, in TestOneInput
    numeral.in_words(fdp.ConsumeInt(len(data)))
  File "/pytils/numeral.py", line 306, in in_words
    check_positive(amount)
  File "/pytils/utils.py", line 40, in check_positive
    raise ValueError("Value must be positive or zero, not %s" % str(value))
```

Comment (xansec) [reply Mayhemfile:1]: And add exec permissions

**Pull request merged**